src/components/Clock.scss

```
@@ -0,0 +1,22 @@
+.Clock {
+  padding: 10px;
+
+  font-size: 80px;
+  text-align: center;
+  color:#f5f5f5;
+  text-shadow:0px 0px 3px #000;-webkit-animation:color 1s infinite;animation:color 7s infinite;
```

Why do you add different animations based on vendor?
```suggestion
  text-shadow: 0px 0px 3px #000;
  animation: color 7s infinite;
```

---

this one